### DISCUSSION TITLE
[Art competition] Bottom-up search

### DISCUSSION THREAD

**Discussion by creationyun** [prosyslab-classroom/cs424-program-reasoning#255] [Art competition]
![DALL·E 2022-11-30 19 52 20 - The various colors diffuse in water](https://user-images.githubusercontent.com/36063812/204779126-551c8239-4bf3-4103-9a16-11a4fba41b55.png)

## Short Description
For me, [*bottom-up search*] means "[*The various colors diffuse in water*]".

## Long Description (up to 3 sentences)
Bottom-up search is a search technique that starts from the first terminal and is combined with each other. This is as if paints of various colors are diffused into each other in the water, and various colors are mixed in various ways to create a beautiful appearance. Expressing colorful colors by combining various colors with pre-determined terminal colors eventually results in a single color, even if it is diffused unorderly.

**Comment by KihongHeo** [comment]
Interesting.

Your art reminds me of [this Youtube video](https://www.youtube.com/watch?v=sxmE3rKo104) I watched last week. Can we make use of such germs for program synthesis? (biocomputing or bio search)
